composer.json

```diff
@@ -63,8 +63,8 @@
     "require-dev": {
         "symfony/phpunit-bridge": "~2.7",
         "doctrine/data-fixtures": "1.0.*",
-        "doctrine/dbal": "~2.2",
-        "doctrine/orm": "~2.2,>=2.2.3",
+        "doctrine/dbal": "~2.4",
+        "doctrine/orm": "~2.4,>=2.4.5",
```

Comment:
this would be the first use in the code base. I'd prefer not doing it in this PR